cmd/internal/shared/config.go

```diff
@@ -76,9 +76,12 @@ var DefaultGRPCConfig = config.GRPC{
 
 // DefaultRedisConfig is the default config for Redis.
 var DefaultRedisConfig = config.Redis{
-	Address:   "localhost:6379",
-	Database:  0,
-	Namespace: []string{"ttn", "v3"},
+	Address:            "localhost:6379",
+	Database:           0,
+	Namespace:          []string{"ttn", "v3"},
+	EnableFailover:     false,
+	FailoverMasterName: "mymaster",
+	FailoverAddresses:  []string{},
```

Comment:
nit; `nil` is a bit cleaner

Reply:
Removed default empty list value as well